### DISCUSSION TITLE
Will restarting manager.py restart the plots thats ongoing?

### DISCUSSION THREAD

**Discussion by VanQz** [swar/Swar-Chia-Plot-Manager#126] [Q&A] [answered]
I have to restart manager.py to load the modified config but i already have plots creating. will restarting the manager stop the current plot creation and more plots will be created when i start again?

**Comment by raid5** [comment, marked as answer]
Nope, restarting the manager will keep the existing plots running until they finish, but new plots that are started by the manager will use the new config settings.
